test/test_operations.py

```diff
@@ -2076,6 +2076,29 @@ def test_multi_init_xla_backend(self):
     self.assertEqual(met.counter_value("RegisterXLAFunctions"), 1)
 
 
+class TestLoweringContext(test_utils.XlaTestCase):
+
+  def test_api(self):
+    device = xm.xla_device()
+    example = torch.tensor([1.0, 2.0, 3.0, 4.0], device=device)
+
+    def network(x):
+      return x + 2.0
+
+    result = network(example)
+
+    ctx = torch_xla._XLAC.lowering.LoweringContext()
+    ctx.build([result])
+    hlo = ctx.hlo()
```

Comment:
is this line needed?

Reply:
It's just there to test whether the API would fail or not. The returned results is a binary and thus cannot be checked.

Reply:
I see.  is it to test `ctx.hlo()` doesn't throw an error